### PR TITLE
Feature/style home page

### PR DESCRIPTION
**Pull request by MTCawthray**
# Thank you for pushing your code!
## What (if any) features are you implementing?
- This branch adds CSS styling to the home page. These stylings include correctly sizing the world map image, creating a nicer looking dropdown that adds to the overall user experience, styling the <h1> to a nice simple font.
- There were several structural changes that happened on an earlier branch that accidentally got merged to main. We decided to submit those details on this branch in order to draw attention to these changes and hopefully get feedback on the most up-to-date version of our application- please see the next section for details.

## What (if anything) did you refactor?
- The App component and the Map component have been refactored into one. Now, all of Map's state, functions, and DOM elements are included in App. 
- We decided to do this in order to make our React-router routes easier to understand and render. We believe that this will make our error handling more streamlined as a result.

## Were there any issues that arose?
- The aforementioned accidental merging of our feature/merge-app-with-map branch into main.

## Is there anything that you need from your teammate?
- NA

## Any other comments, questions, or concerns?
- These styles are subject to change depending on our extra iterations. Please feel free to let me know if there is anything you would like adjusted in the design. I tried to keep it as simple as possible to not distract from the user experience. 

## Type of change
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## How Has This Been Tested?
- [x] open index.html
- [ ] console.log()
- [x] dev tools

## Checklist:
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -24,7 +24,10 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
-    <title>React App</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
```
Comment (maeduphorne) [inline public/index.html:27]: Fonts look great! So glad we are getting started on a little bit of styling

File: src/components/App/App.tsx [hunk 2]
```diff
@@ -55,7 +55,7 @@ function App() {
       apiCalls.fetchCountriesData()
       .then((data) => setCountries(data))
     }
-    
+   // try using an empty array on line 59 after the closing curly. This may let us get rid of the conditional!
```
Comment (maeduphorne) [inline src/components/App/App.tsx:58]: this was one of my takeaways from the hooks lesson today as well! We should ask instructors if the conditional is the best way to use useEffect here to solve the infinite loop issue

**Review by maeduphorne — COMMENTED** [review]
Thanks for pushing! Not a lot to review here, but the mistaken merge set us up with a lot of new elements in our app. Router is working - big win!

File: src/components/App/App.tsx [hunk 3]
```diff
@@ -55,7 +55,7 @@ function App() {
       apiCalls.fetchCountriesData()
```
Comment (kaylagordon) [inline src/components/App/App.tsx:55]: - What is the purpose of the fragment (`<>`) that wraps the `form` element?
- Remove console logs before pushing code into the main branch
-  The "DONE"/"TO DO" list in your `App.js` file is more appropriate to put in your PR, project board, and/or issues. Keep that out of your code.

**Pull request merged**